components/style/logical_geometry.rs

```diff
@@ -215,6 +215,7 @@ impl Debug for DebugWritingMode {
 
 // Used to specify the logical direction.
 #[derive(Debug, Clone, Copy, PartialEq)]
+#[cfg_attr(feature = "servo", derive(Serialize))]
```

Comment:
you'd need to use a build script with serde_codegen to make this work. I don't think we need this for stylo now, so we can just leave it out I guess.